tests/schema_update_test.go

```diff
@@ -1536,6 +1546,11 @@ func TestVCS_SQL_Review(t *testing.T) {
 
 			// Create a repository.
 			ctl.vcsProvider.CreateRepository(test.externalID)
+
+			// Create the branch.
+			err = ctl.vcsProvider.CreateBranch(test.externalID, "feature/foo")
```

Comment:
What about unhappy case?

Reply:
What does unhappy case mean here?

Reply:
branch not found

Reply:
The *branch not found* case is going to be handled at https://github.com/bytebase/bytebase/pull/3396.
As for this PR, we turn to do "create a branch" and "bind vcs configuration with a branch" separately when writing integration tests, to leave some space for new tests that will focus on the *branch not found* cases.

Reply:
Gotcha.